deepcave/runs/__init__.py

```diff
@@ -893,7 +892,7 @@ def get_encoded_data(
 
 def check_equality(
     runs: List[AbstractRun],
-    meta: bool = True,
+    meta: bool = False,
```

Comment:
Is this now always false or just for the smac 2 converter?

Reply:
No, this way it is false for all the converters. Another option could be to check if the run is a SMAC2Run and only then leave out the meta check.